conda/cli/main_env.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+"""Entry point for all conda-env subcommands."""
+from argparse import ArgumentParser
+from importlib import import_module
+
+# pip_util.py import on_win from conda.exports
+# conda.exports resets the context
+# we need to import conda.exports here so that the context is not lost
+# when importing pip (and pip_util)
+import conda.exports  # noqa
+from conda.base.context import context
+
+from . import (
+    main_env_config,
+    main_env_create,
+    main_env_export,
+    main_env_list,
+    main_env_remove,
+    main_env_update,
+)
+from .helpers import show_help_on_empty_command
+
+
+def configure_parser():
```

Comment:
This isn't integrated into conda's `ArgumentParser`, need to import in `conda.cli.conda_argparse` and include in `generate_parser`